### PR TITLE
Fix: typos

### PR DESCRIPTION
**Pull request by omahs**
Fix: typos

### CODE REVIEW

File: docs/python_vm/README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Functions:
...
 * [`compute_res(instruction, op0, op1)`](https://github.com/starkware-libs/cairo-lang/blob/b614d1867c64f3fb2cf4a4879348cfcf87c3a5a7/src/starkware/cairo/lang/vm/vm_core.py#L241) -> Optional(MaybeRelocatable): Returns computed res based on instruction.res: returns op1 if OP1, op1+op2 mod prime if ADD, op0 * op1 mod prime if MUL (and both operands are not relocatable), none if UNCONSTRAINED (handled elsewhere???, should be inverse of dst), or fails otherwise
-* [`compute_operands(instruction) -> Tuple(Operands, List(int))`](#computeop) -> Returns the Operands(name is plural, its one) based on the Instruction, coputes dst, op0 and op1 adresses, deduces op0 and op1, validates and updates dst, op0, op1.
+* [`compute_operands(instruction) -> Tuple(Operands, List(int))`](#computeop) -> Returns the Operands(name is plural, its one) based on the Instruction, coputes dst, op0 and op1 addresses, deduces op0 and op1, validates and updates dst, op0, op1.
```
Comment (MegaRedHand) [inline docs/python_vm/README.md:70]:
```suggestion
* [`compute_operands(instruction) -> Tuple(Operands, List(int))`](#computeop) -> Returns the Operands(name is plural, its one) based on the Instruction, computes dst, op0 and op1 addresses, deduces op0 and op1, validates and updates dst, op0, op1.
```

File: docs/python_vm/README.md [hunk 2]
```diff
@@ -57,8 +57,8 @@ The [VirtualMachine](https://github.com/starkware-libs/cairo-lang/blob/b614d1867
 * `hint_locals` : (Dict)
 * `static_locals` : (Optional Dict)
 VirtualMachineBase's init is used to set these values (plus other ones), the next ones are exclusive to the VirtualMachine:
-* `accessed_addresses` : (Set that keeps track of memory adresses accessed by cairo instructions)
-* `trace` : (List of TraceEntry, that each contain the run_context's pc, ap and fp at that moment. A TraceEntry is added after every instruction (Before update_registers is called))
+* `accessed_addresses` : (Set that keeps track of memory addresses accessed by cairo instructions)
+* `trace` : (List of TraceEntry, that each contains the run_context's pc, ap and fp at that moment. A TraceEntry is added after every instruction (Before update_registers is called))
```
Comment (MegaRedHand) [inline docs/python_vm/README.md:61]:
```suggestion
* `trace` : (List of `TraceEntry`s, each containing the run_context's pc, ap, and fp at that moment. A `TraceEntry` is added after every instruction (Before update_registers is called))
```

File: docs/python_vm/README.md [hunk 3]
```diff
@@ -163,10 +163,10 @@ An `IdentifierManager` has a `root` (an `IdentifierScope`) and a `dict` ( a Muta
...
 
-A `ReferenceManager` containsa list of `Reference`, and the methods to add a reference and return ist position (`alloc_id`), and to get a reference by its position (`get_ref`)
-A `Reference` is a reference to a memory adress for a specific location in the program (This is the reference that is created when you use `let`). It contains`pc` (int), `value` (Expression), `ap_tracking_data` (`RegTrackingData`), `locations` (A list of `Location`, contains a list of definition sites from the reference, it will contain multiple locations when the reference is defined from the convergence of multiple reference definitions), `definition_code_element` (an optional `CodeElement`, the code element that created this reference).
+A `ReferenceManager` contains a list of `Reference`, and the methods to add a reference and return its position (`alloc_id`), and to get a reference by its position (`get_ref`)
```
Comment (MegaRedHand) [inline docs/python_vm/README.md:166]:
```suggestion
A `ReferenceManager` contains a list of `Reference`s, and methods to add a reference while returning its position (`alloc_id`), and to get a reference by its position (`get_ref`)
```

File: docs/python_vm/README.md [hunk 4]
```diff
@@ -396,7 +396,7 @@ The compiled hints are obtained by using python's [`compile`](https://github.com
...
 The current steps are taken to execute hints with each iteration:
 1. We obtain the list of the hints at our current pc (if any) and iterate over it
-2. We create an `exec_locals` dictionary that will contain our program’s input, memory (validated_memory), current registers (ap, fp, pc), the current step, ids which contains memory, ap, fp and pc as constants, the functions load_program, enter_scope and exit_scope, and the static_locals. The static_locals add to the exec_locals the program’s prime, the MemorySegmentManager (for example, this is used by alloc to add a memory segment for a undetermined-lengh array), and basic operations such as fadd, fsub, fdiv, etc.
+2. We create an `exec_locals` dictionary that will contain our program’s input, memory (validated_memory), current registers (ap, fp, pc), the current step, ids which contains memory, ap, fp and pc as constants, the functions load_program, enter_scope and exit_scope, and the static_locals. The static_locals add to the exec_locals the program’s prime, the MemorySegmentManager (for example, this is used by alloc to add a memory segment for an undetermined-lengh array), and basic operations such as fadd, fsub, fdiv, etc.
```
Comment (MegaRedHand) [inline docs/python_vm/README.md:399]:
```suggestion
2. We create an `exec_locals` dictionary that will contain our program’s input, memory (validated_memory), current registers (ap, fp, pc), the current step, ids (which contains memory), ap, fp, and pc as constants, the functions load_program, enter_scope and exit_scope, and the static_locals. The static_locals add to the exec_locals the program’s prime, the MemorySegmentManager (for example, this is used by alloc to add a memory segment for an undetermined-length array), and basic operations such as fadd, fsub, fdiv, etc.
```

**Comment by omahs** [comment]
@MegaRedHand Thanks for the quick review! I included the changes

**Review by MegaRedHand — APPROVED** [review]
👍

**Review by pefontana — APPROVED** [review]

**Review by pefontana — APPROVED** [review]
Thanks for the contribution @omahs !

**Pull request merged**